packages/psss/src/utils/PersistGateProvider.js

```diff
@@ -8,23 +8,21 @@ import { PersistGate } from 'redux-persist/integration/react';
 import PropTypes from 'prop-types';
 
 function initPersistor(store) {
-  const persistor = persistStore(store, null);
+  const persistor = persistStore(store);
 
-  // if you run into problems with redux state, call "purge()" in the dev console
```

Comment:
This was never being used and was a bit distracting in the code so I removed it.